src/Compilers/Test/Utilities/CSharp/CompilationTestUtils.cs

```diff
@@ -307,15 +307,16 @@ internal static void VerifyTypes(this CSharpCompilation compilation, SyntaxTree
             foreach (var annotations in annotationsByMethod)
             {
                 var method = (MethodSymbol)model.GetDeclaredSymbol(annotations.Key);
+                var dictionary = new Dictionary<SyntaxNode, TypeSymbolWithAnnotations>();
```

Comment:
>dictionary [](start = 20, length = 10)

nit: definition can be moved closer to usage. Maybe have this done by `var dictionary = TopLevelNullabilityRetriever.BuildMap(rewritten);`?